### ISSUE TITLE
NuGet package failed to install

### ISSUE THREAD

**Issue by D3MaxT** [aspnet/LibraryManager#56]
Testing Visual Studio 2017 15.7 Preview 3 release.
Getting an error message: "NuGet package failed to install" when trying to enable the "Enable Restore on Build" option (right-click on the libman.json file and click on the "Enable Restore on Build" option).

Any ideas why?

**Comment by raquelsa** [comment]
We are looking in this issue now. I will update you once it is fixed. Thanks for reporting!

**Comment by D3MaxT** [comment]
I guess it wasn't published to NuGet yet at the time.
Looks like it is there now: Microsoft.Web.LibraryManager.Build 1.0.20

It's working now.

Thank you.

**Comment by D3MaxT** [comment]
![sfnuget](https://user-images.githubusercontent.com/24484920/38622985-1eb94696-3d6a-11e8-8dae-16a2a06aa87a.jpg)

On my work PC the error was still happening. I tracked it down to Service Fabric registering a Machine-wide NuGet package source. I have actually uninstalled Service Fabric on this PC, but it kept the NuGet setting in Visual Studio for NuGet package sources. I unchecked the box and it fixed the problem on the work PC as well.

This is more of a Visual Studio or Service Fabrci bug. Needs to be passed along to the right team.
